src/config.rs

```diff
@@ -146,11 +146,43 @@ pub struct Config {
     pub slack_token: Option<String>,
     #[clap(
         long,
-        value_name = "SLACK_WEBHOOK",
-        help = "Webhook to the Slack bot App",
-        env = "SLACK_WEBHOOK"
+        value_name = "SLACK_CHANNEL",
+        help = "Name of Slack channel to send messages to",
```

Comment:
iirc the slack bot requires the channel to be public? Maybe add that in help?

Reply:
good suggestion, yeah it has to be public